src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -443,10 +444,11 @@ public function groupTranslations($results)
 
             $result = [];
             foreach ($grouped->combine('field', 'content', 'locale') as $locale => $keys) {
-                $translation = new Entity($keys + ['locale' => $locale], [
+                $translation = $this->_table->newEntity($keys + ['locale' => $locale], [
                     'markNew' => false,
                     'useSetters' => false,
-                    'markClean' => true
+                    'markClean' => true,
```

Comment:
None of the options above is passed to the entity constructor when using newEntity.

A better idea is to the `$table->entityClass()` and instantiate that